src/core/Slider/Slider.tsx

```diff
@@ -0,0 +1,638 @@
+const Thumb = ({ index, isActive }: { index: number; isActive: boolean }) => {
+  const {
+    hideTooltip,
+    railDomain,
+    setThumbMoveIndex,
+    toolTipFunction,
+    step,
+    values,
+    setThumbValue,
+    thumbMode,
+    toolTipPosition,
+  } = useSliderContext();
+  const thumbRef = React.useRef<HTMLDivElement>(null);
+  const handleOnKeyDown = React.useCallback(
+    (event: React.KeyboardEvent<HTMLDivElement>) => {
+      const [minVal, maxVal] = getAllowableThumbRange(
+        thumbMode,
+        values,
+        index,
+        railDomain,
+        step,
+      );
+
+      if (event.key === 'ArrowLeft') {
+        const newValue = Math.max(values[index] - step, minVal);
+        setThumbValue(index, newValue);
+      } else if (event.key === 'ArrowRight') {
+        const newValue = Math.min(values[index] + step, maxVal);
+        setThumbValue(index, newValue);
+      } else if (event.key === 'Home') {
+        setThumbValue(index, minVal);
+      } else if (event.key === 'End') {
+        setThumbValue(index, maxVal);
+      }
+    },
+    [thumbMode, values, index, railDomain, step, setThumbValue],
+  );
+
+  const handlePointerDownOnThumb = React.useCallback(
+    (event: React.PointerEvent) => {
+      thumbRef.current?.focus();
+      setThumbMoveIndex(index);
+      event.preventDefault();
+      event.stopPropagation();
+    },
+    [index, setThumbMoveIndex],
+  );
+
+  const [hasFocus, setHasFocus] = React.useState(false);
+  const [isHovered, setIsHovered] = React.useState(false);
+
+  const generateTooltip = React.useCallback(
+    (val: number): string => {
+      return toolTipFunction
+        ? toolTipFunction(val, step)
+        : formatNumberValue(val, step);
+    },
+    [toolTipFunction, step],
+  );
+
+  const currentValue = values[index];
+  const tooltipContent = generateTooltip(currentValue);
+  const leftPercent =
+    (100.0 * (currentValue - railDomain.min)) /
+    (railDomain.max - railDomain.min);
+
+  const thumbClassNames = React.useMemo(
+    () => cx('iui-slider-thumb', isActive && 'iui-active'),
+    [isActive],
+  );
+  const showTooltip = !hideTooltip && (isActive || hasFocus || isHovered);
+  return (
+    <Tooltip
+      visible={showTooltip}
+      content={tooltipContent}
+      placement={toolTipPosition}
+    >
+      <div
+        ref={thumbRef}
+        style={{ left: `${leftPercent}%` }}
+        className={thumbClassNames}
+        role='slider'
+        tabIndex={0}
+        aria-valuemin={railDomain.min}
+        aria-valuenow={railDomain.max}
+        aria-valuemax={currentValue}
+        onPointerDown={handlePointerDownOnThumb}
+        onKeyDown={handleOnKeyDown}
+        onFocus={() => setHasFocus(true)}
+        onBlur={() => setHasFocus(false)}
+        onMouseEnter={() => setIsHovered(true)}
+        onMouseLeave={() => setIsHovered(false)}
+      />
+    </Tooltip>
+  );
+};
+
+/** Properties for [[Slider]] component */
```

Comment:
Is this some special syntax in jsdoc? [[Slider]] Does not show anything specific...

Reply:
In iModeljs repo this generate a link to the Slider component when the API docs are generated.  If it does not do that for iTwinUI I can remove them.

Reply:
oh, interesting..
We dont have such a thing in iTwinUI